packages/shared/src/components/cards/FlaggedContainer.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactElement, ReactNode } from 'react';
+import styles from './Card.module.css';
+import { RaisedLabel, RaisedLabelProps, RaisedLabelType } from './RaisedLabel';
+
+interface FlaggedContainerProps {
+  children: ReactNode;
+  labelProps: RaisedLabelProps;
+}
+
+export const getFlaggedContainer = (
+  children: ReactNode,
+  props: Partial<RaisedLabelProps> = {},
+): ReactElement => (
+  <div className={`relative ${styles.cardContainer}`}>
+    {children}
+    <RaisedLabel type={RaisedLabelType.Pinned} {...props} />
+  </div>
+);
+
+const FlaggedContainer = ({
```

Comment:
Where do you use this?
I think you can just delete this whole file actually.

Reply:
We are still using the `getFlaggedContainer`. Not anywhere atm but kept it there if anyone needs it. But if you want we can definitely remove this one.